src/trel_dnssd/trel_dnssd.cpp

```diff
@@ -297,6 +297,12 @@ void TrelDnssd::OnTrelServiceInstanceAdded(const Mdns::Publisher::DiscoveredInst
     {
         otbrLogDebug("Peer address: %s", addr.ToString().c_str());
 
+        // Skip anycast (Refer to https://datatracker.ietf.org/doc/html/rfc2373#section-2.6.1)
+        if (addr.m64[1] == 0)
```

Comment:
>```
>   |                         n bits                 |   128-n bits   |
>   +------------------------------------------------+----------------+
>   |                   subnet prefix                | 00000000000000 |
>   +------------------------------------------------+----------------+
>```

We are assuming the subnet prefix `n` will be `<= 64` (so the `m64[1]` will zero). Is `n>64` possible?

Reply:
Generally on-link prefix allocated are of len 64 I think. we may have this assumption for now and take care of it if it happens somehow?